### PR TITLE
chore: remove duplicate gountries iniatialization and add more test

### PR DESCRIPTION
**Pull request by totokherlambang043**
fix: update gountries initialization

### CODE REVIEW

**Comment by thoriqsatriya** [comment]
look good to me

File: rules.go [hunk 1]
```diff
@@ -104,7 +103,11 @@ func IsContainWhitespace(str string) bool {
 }
 
 func IsCountry(str string) bool {
-	_, err := gountries.New().FindCountryByAlpha(str)
+	var err error
+	if gountriesQuery != nil {
```
Comment (thoriqsatriya) [inline rules.go:107]: if gountriesQuery nil, this func will always return `true`, right? I think if gountriesQuery is nil, we should consider it as an error, so it always return false. 
Or we can also init here if gountriesQuery is nil. What do you think?

Comment (anggorodewanto) [reply rules.go:107]: yes, lazy init, i like that

Comment (totokherlambang043) [reply rules.go:107]: sure, let me add lazy initialization.

**Review by thoriqsatriya — APPROVED** [review]

**Pull request merged**